src/Orchard.Cms.Web/Modules/Orchard.Commons/Orchard.Commons.csproj

```diff
@@ -1,16 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="..\..\..\..\build\common.props" />
```

Comment:
Yes, in the `packages` branch, in the `Orchard.Cms.Web` directory, we put a `Directory.Build.props` file which is used (because of its name) in all sub directories (Modules and Themes).

Then in this file we can import other .props and .targets stored in any other place in the repo.